xil_carma_cloud/cdasim_config/cdasim/logback.xml

```diff
@@ -175,37 +175,37 @@
                 <appender-ref ref="CellLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.application.ambassador.navigation" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.application.ambassador.navigation" additivity="false" level="INFO">
                 <appender-ref ref="NavigationLog"/>
             </logger>
-            <logger name="org.eclipse.mosaic.lib.routing" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.lib.routing" additivity="false" level="INFO">
                 <appender-ref ref="NavigationLog"/>
             </logger>
-            <logger name="com.graphhopper" additivity="false" level="TRACE">
+            <logger name="com.graphhopper" additivity="false" level="INFO">
                 <appender-ref ref="NavigationLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.mapping" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.mapping" additivity="false" level="INFO">
                 <appender-ref ref="MappingLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.environment" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.environment" additivity="false" level="INFO">
                 <appender-ref ref="EnvironmentLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.sumo" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.sumo" additivity="false" level="INFO">
                 <appender-ref ref="TrafficLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.carla" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.carla" additivity="false" level="INFO">
                 <appender-ref ref="CarlaLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.carma" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.carma" additivity="false" level="INFO">
                 <appender-ref ref="CarmaLog"/>
             </logger>
 
-            <logger name="org.eclipse.mosaic.fed.infrastructure" additivity="false" level="TRACE">
+            <logger name="org.eclipse.mosaic.fed.infrastructure" additivity="false" level="INFO">
```

Comment:
Could we keep the log level for infrastructure, CARMA, and ns-3 set to TRACE for now, and revert it back to INFO when we are ready to release the CARMA-Cloud integration? Some logs are necessary for the verification test to track down the message ID. We have set these logs to "debug" because setting them to "info" would generate too many lines, which would slow down the system and occupy too much disk space during normal operation.

Reply:
Sure I would prefer we update our testing procedure to depend on easier accessible and more robust sources of information rather than debug logs. For example we should be looking at service UI  components to confirm messaging. This can be saved for the future but should be something we are working towards now.

Reply:
I am changing the infrastructure and ns3 loggers back to trace and debug respectively. I am leaving the others at info. Please let me know which other log configurations would be necessary.